app/javascript/components/App.js

```diff
@@ -23,10 +29,19 @@ class App extends React.Component {
   console.log("sign_out_route:", sign_out_route)
     return (
         <>
-           <Home />
+          <Header />
```

Comment:
All the Devise content should be removed from App.js and passed into your Header component.